docs/SlavaUkraini/dialog.js

```diff
@@ -0,0 +1,44 @@
+var SESSION_KEY = 'ukraini-dialog-session';
+var ONE_DAY_MILLI_SEC = 24 * 60 * 60 * 1000;
+
+function openSlavaUkrainiDialog() {
+
+	if (sessionStorage) {
+		var session = sessionStorage.getItem(SESSION_KEY);
+		// open the dialog only every 24 hours
+		if (session && Date.now() - session < ONE_DAY_MILLI_SEC) {
+			return;
+		}
+	}
+
+	var html = '<div class=\'container iframe-container\'><iframe tabindex="1" src=\'/SlavaUkraini/\'></iframe><a tabindex="2" role="button" class=\'close-dialog\' aria-label=\'close\'>&times;</a></div>';
+
+	var dialog = document.createElement('div');
+	dialog.id = 'ukraini-dialog';
+	dialog.innerHTML = html;
+
+	document.body.appendChild(dialog);

```

Comment:
Also, the logical order of the dialog is to be the first child of body since it's the first content presented to the user, although it matters less if it's autofocused.